.github/workflows/core.yml

```diff
@@ -4,15 +4,21 @@ on:
   pull_request:
     paths:
     - 'FirebaseCore**'
+    - '.github/workflows/core.yml'
+  schedule:
+    # Run every day at 1am (PST) - cron uses UTC times
+    - cron:  '0 7 * * *'
```

Comment:
I wonder is there a way to move common parameters to kind of a common parent config?

Reply:
Maybe? I'll investigate in a future PR.

Reply:
Sure, it's my curiosity mostly and is nice to have but doesn't look required.